ci-operator/step-registry/agent/devscripts/gather/agent-devscripts-gather-commands.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ agent gather command ************"
+echo "### Gathering logs..."
+timeout -s 9 5m ssh "${SSHOPTS[@]}" "root@${IP}" bash - <<EOF
```

Comment:
`SSHOPTS` and `IP` are set only by sourcing the `packet-conf.sh` file, which is currently generated by the [packet-setup step](https://github.com/openshift/release/blob/0fc3adfbd284c6fda81fff80dd903b605ceca4e3/ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh#L70-L93), ie: 

```
# Fetch packet basic configuration
# shellcheck source=/dev/null
source "${SHARED_DIR}/packet-conf.sh"
```